internal/provider/resource_setting_mgmt_test.go

```diff
@@ -1,13 +1,23 @@
 package provider
 
 import (
+	"sync"
 	"testing"
 
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
 )
 
+var settingMgmtLock = sync.Mutex{}
+
 func TestAccSettingMgmt_basic(t *testing.T) {
 	resource.ParallelTest(t, resource.TestCase{
+		PreCheck: func() {
+			preCheck(t)
+			settingMgmtLock.Lock()
```

Comment:
Is this equivalent to just not using `ParralelTest`?

Reply:
I thought about just changing it but its not quite the same, because `ParallelTest` is parallel across the whole package, `Test` is sequential across the whole package. We'd still want this to run in parallel with other non setting tests.